ppa/archive/views.py

```diff
@@ -166,3 +171,72 @@ def render_to_csv(self, data):
 
     def get(self, *args, **kwargs):
         return self.render_to_csv(self.get_data())
+
+
+class AddToCollection(ListView, FormMixin, ProcessFormView):
+    '''
+    View to bulk add a queryset of :class:`ppa.archive.models.DigitizedWork`
+    to a set of :class:`ppa.archive.models.Collection instances`.
+
```

Comment:
Should document here that it expects a list of DigitizedWork ids to be set in the session.